rnd/market/Dockerfile

```diff
@@ -25,19 +26,31 @@ ENV POETRY_VERSION=1.8.3 \
     PATH="$POETRY_HOME/bin:$PATH"
 RUN pip3 install poetry
 
-COPY rnd/market /app/rnd/market
+FROM server_base AS server_dependencies
+
+RUN mkdir -p /app/autogpt
+RUN mkdir -p /app/forge
+RUN mkdir -p /app/rnd/autogpt_libs
+RUN mkdir -p /app/rnd/market
```

Comment:
Docker automatically creates folders when you copy stuff to them, so there's no need for this